companyweb_base/models/partner_cweb_model.py

```diff
@@ -0,0 +1,527 @@
+    cweb_image_enable = fields.Boolean(
+        "CompanyWeb Barometer Image Enabled", readonly=True
+    )
+
+    cweb_warnings = fields.Text("CompanyWeb Warnings", readonly=True)
+    cweb_warnings_enable = fields.Boolean("CompanyWeb Warnings Enabled", readonly=True)
+    cweb_url = fields.Char("CompanyWeb Detailed Report", readonly=True)
+    cweb_url_enable = fields.Boolean(
+        "CompanyWeb Detailed Report Enabled", readonly=True
+    )
+    cweb_url_report = fields.Char("CompanyWeb Url Report", readonly=True)
+    cweb_url_report_enable = fields.Boolean(
+        "CompanyWeb Url Report Enabled", readonly=True
+    )
+    cweb_vat_liable = fields.Boolean("CompanyWeb Subject to VAT", readonly=True)
+    cweb_vat_liable_enable = fields.Boolean(
+        "CompanyWeb Subject to VAT Enabled", readonly=True
+    )
+    cweb_balance_data_enable = fields.Boolean(
+        "CompanyWeb Balance Data Enabled", readonly=True
+    )
+    cweb_balance_year = fields.Char("CompanyWeb Balance Year", readonly=True)
+    cweb_balance_year_enable = fields.Boolean(
+        "CompanyWeb Balance Year Enabled", readonly=True
+    )
+    cweb_closed_date = fields.Date("CompanyWeb Closed Date", readonly=True)
+    cweb_closed_date_enable = fields.Boolean(
+        "CompanyWeb Closed Date Enabled", readonly=True
+    )
+    cweb_equityCapital = fields.Float("CompanyWeb Equity Capital", readonly=True)
+    cweb_equityCapital_enable = fields.Boolean(
+        "CompanyWeb Equity Capital Enabled", readonly=True
+    )
+    cweb_average_fte = fields.Float(
+        "CompanyWeb Average number of staff in FTE", readonly=True
+    )
+    cweb_average_fte_enable = fields.Boolean(
+        "CompanyWeb Average number of staff in FTE Enabled", readonly=True
+    )
+    cweb_addedValue = fields.Float("CompanyWeb Gross Margin (+/-)", readonly=True)
+    cweb_addedValue_enable = fields.Boolean(
+        "CompanyWeb Gross Margin (+/-) Enabled", readonly=True
+    )
+    cweb_turnover = fields.Float("CompanyWeb Turnover", readonly=True)
+    cweb_turnover_enable = fields.Boolean("CompanyWeb Turnover Enabled", readonly=True)
+    cweb_result = fields.Float(
+        "CompanyWeb Fiscal Year Profit/Loss (+/-)", readonly=True
+    )
+    cweb_result_enable = fields.Boolean(
+        "CompanyWeb Fiscal Year Profit/Loss (+/-) Enabled", readonly=True
+    )
+    cweb_prefLang = fields.Many2one(
+        "res.lang", string="CompanyWeb Preferred Language", readonly=True
+    )
+    cweb_prefLang_enable = fields.Boolean(
+        "CompanyWeb Preferred Language Enabled", readonly=True
+    )
+
+    cweb_show_button_enhance = fields.Boolean(
+        "CompanyWeb Button Enhance Enabled", compute="_compute_show_button_enhance"
+    )
+    cweb_show_button_address = fields.Boolean(
+        "CompanyWeb Button Address Enabled", compute="_compute_show_button_address"
+    )
+    cweb_show_tab = fields.Boolean(
+        "CompanyWeb Tab Enabled", compute="_compute_show_tab"
```

Comment:
```suggestion
        "CompanyWeb Tab Enabled", compute="_compute_cweb_show_tab"
```

(convention for compute method names)